geos/bld.bat

```diff
@@ -0,0 +1,11 @@
+cmake -G "NMake Makefiles" -DCMAKE_PREFIX_PATH=%PREFIX% -DCMAKE_INSTALL_PREFIX:PATH=%PREFIX% -D CMAKE_BUILD_TYPE=Release .
+if errorlevel 1 exit 1
+
+nmake
+if errorlevel 1 exit 1
+
+nmake install
+if errorlevel 1 exit 1
+
+move %PREFIX%\bin\*.* %PREFIX%
```

Comment:
There is no need to move the dll files to `%PREFIX%`.  They need to stay in bin.